### DISCUSSION TITLE
Trouble Installing Beta on Ryujinx

### DISCUSSION THREAD

**Discussion by culturedminecrafter** [jugeeya/UltimateTrainingModpack#457] [Q&A]
I downloaded the atmosphere folder and pasted the folders inside of contents into AppData\Roaming\Ryujinx\mods\contents\01006a800016e000, but whenever I open training mode it crashes. This doesn't happen in regular games, only in training mode. I'm not really sure what the issue is, could someone help me out?

**Comment by jugeeya** [comment]
Is your Smash version 13.0.1? That's the only reason it would crash really.
